styleguide/_themes/derek/scss/components/_ceiling.scss

```diff
@@ -1,143 +1,141 @@
+      a {
+        font-size: 13px;
       }
     }
+  }
 
-    .dropdown-menu {
-      padding: 0;
-      min-width:0px;
-      li a{
-        padding: 3px floor($ceiling-margin * 2);
-      }
+  .menu-inline {
+    @extend %menu-inline;
+  }
+}
+
+%menu-inline {
+  list-style: none;
+  margin-bottom: 0;
+  margin-left: -5px;
+  padding-left: 0;
+
+  > li {
+    border-right: 1px solid $darkest-gray;
+    display: inline-block;
+    margin: floor($ceiling-margin + 2) 0;
+    padding: 0 floor($ceiling-margin + 2);
+
+    &.no-border,
+    &:last-child {
+      border: 0;
     }
+  }
 
-    .dropdown{
-      padding: 0;
-      &:after {
-        content: "";
-      }
-      ul, li, a {
-        margin: 0;
-        background-color: $medium-gray;
-        @extend .white;
-        &:hover, &:focus, &:active {
-          background-color: lighten($medium-gray, 5%);
-        }
-      }
-      > a {
-        @extend .btn;
-        @extend .btn-default;
-        background-color: $medium-gray;
-        font-size: $ceiling-font-size;
-        border: none;
-        border-radius: 0px;
-        &:hover, &:focus, &:active {
-          background-color: lighten($medium-gray, 5%);
-          @extend .white;
-        }
+  .dropdown-menu {
+    min-width: 0;
+    padding: 0;
+
+    li {
+      a {
+        padding: 3px floor($ceiling-margin * 2);
       }
     }
+  }
 
+  .dropdown {
+    padding: 0;
 
+    &:after {
+      content: '';
+    }
 
-    .dropdown > .dropdown-green {
-      background-color: $signup-green;
-      @extend .white;
-      ul, li, a {
-        background-color: $signup-green;
-        &:hover, &:focus, &:active {
-          background-color: lighten($signup-green, 5%);
-          @extend .white;
+    ul,
+    li,
+    a {
+      @extend %white;
+      background-color: $medium-gray;
+      margin: 0;
+
+      &.dropdown-green {
+        background-color: $dropdown-green;
+        a { background-color: $dropdown-green; }
+
+        &:active,
+        &:hover,
+        &:focus {
+          background-color: $dropdown-green-hover;
         }
       }
-      &:hover, &:focus, &:active {
-        background-color: lighten($signup-green, 5%);
-        @extend .white;
+
+      &:hover,
+      &:focus,
+      &:active {
+        background-color: lighten($medium-gray, 5%);
       }
     }
 
-    .dropdown.open > a.dropdown-green {
-      background-color: $signup-green;
-      @extend .white;
-      &:focus, &:active {
-        background-color: lighten($signup-green, 5%);
-        @extend .white;
+    > a {
+      background-color: $medium-gray;
+      border: 0;
+      border-radius: 0;
+      font-size: $ceiling-font-size;
+
+      &:hover {
+        @extend %white;
+        background-color: lighten($medium-gray, 5%);
       }
-      li a{
-        @extend .white;
-        &:hover {
-          background-color: lighten($signup-green, 5%);
-        }
+
+      &:focus,
+      &:active {
+        background-color: $dropdown-light-gray;
```

Comment:
There were many color codes w/o variable names. the added colors are just to round up all those